test/bigtest/network/config.js

```diff
@@ -23,7 +35,17 @@ export default function config() {
     // };
   });
 
-  this.put(`${ORDERS_API}/:id`, noop);
+  this.put(`${ORDERS_API}/:id`, (schema, request) => {
+    const attrs = JSON.parse(request.requestBody);
+
+    return schema.orders.create(attrs);
```

Comment:
I don't think that it should call `create` on `this.put`. Anyway, what if you use a shortcut like `this.put(`${ORDERS_API}/:id`)` ?

Reply:
Unfortunately, shortcut construction doesn't work in this case

Reply:
still opened question for `call create on this.put`

Reply:
Updated. Thanks for help